setup.cfg

```diff
@@ -34,3 +34,6 @@ packages = rtree
 
 [options.package_data]
 rtree = lib
+
+[flake8]
+max-line-length = 88
```

Comment:
Black uses a line length of 88 by default. We could instead change black's line length to 79/80 to match flake8, but I think most projects are simply using what black does these days.

Reply:
I've no strong opinion. The purpose of the linting is to try to prevent the codebase's style from diverging from norms.